### PR TITLE
Fix empty push notifications

### LINKED ISSUE #568: Empty push notifications
**Description**

Since recently, I get empty push notifications - no title or no body. See attached screenshot.

**Steps to Reproduce**

No idea yet but I think it's related to when push notifications get merged since when I click on it, I get redirected to `/notifications`. But there were no changes to the service worker [since we added diagnostics](3a7c3f7) (and #563 wasn't merged yet):

```
$ git log -1 --oneline sw/index.js
3a7c3f7 Add setting to send diagnostics back to SN (#463)
```

And the last changes in `api/webPush/index.js` was 94fbabc; also too long ago since this started to happen recently.

```
$ git log -1 --oneline api/webPush/index.js
94fbabc Notifications for when you are forwarded sats from a post (#467)
```

**Expected behavior**

Not empty push notification

**Screenshots**

signal-2023-10-17-031321

**Environment:**

Happens on Android 9 using Brave and Brave on Linux (iirc)

**Additional context:**

I am surprised no one else reported this already

### PR DESCRIPTION
**Pull request by ekzyis**
Fixes #568

https://github.com/stackernews/stacker.news/assets/27162016/153d4b96-d8c1-4425-9f9a-95513c6e472a

### CODE REVIEW

File: sw/index.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ self.addEventListener('push', async function (event) {
   const { tag } = payload.options
   event.waitUntil((async () => {
     // TIP and EARN notifications simply replace the previous notifications
```
Comment (kevkevinpal) [inline sw/index.js:79]: you might want to update this comment aswell

Comment (ekzyis) [reply sw/index.js:79]: Thought so as well but then I thought forwarded tips are tips as well, sooo... :smile:

**Comment by ekzyis** [comment]
Mhh, this is still happening on https://stacker.news

you can test by enabling push notifications for forwarded sats and then anon zapping an item twice for which sats are forwarded to you.

**Comment by ekzyis** [comment]
Ah, oh, there must have been a rebase error somewhere since the change was lost somehow. [`FORWARDEDTIP` is no longer mentioned in the service worker on `master`](https://github.com/stackernews/stacker.news/blob/0c9131f6988fdf123a03b26bd6abe0a8d198e21a/sw/eventListener.js#L56)

**Pull request merged**